lib/tasks/migrations/plans/glue_load_plans_deprecated.rake

```diff
@@ -0,0 +1,60 @@
+#! This rake is deprecated. Please use the rake `bundle exec rake seed:load_products`
+
+namespace :seed do
+  task :load_plans, [:year] => :environment do |task, args|
```

Comment:
@jacobkagon Can you also rename the task? This way the code will not execute when we run `bundle exec rake seed:load_plans`